index.html

```diff
@@ -168,6 +168,31 @@ <h3>Skills</h3>
         
     </ul>
 </section>
-
+<section>
+    <h3 class="info3">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h3>
+    </section>
+    <section class="footer">
+    <form
+    action="https://formspree.io/f/xbjenlov"
+    method="POST">
+        <ul>
+            <li><input type="text" class="name-email" name="name" maxlength="30" placeholder="Full name" required></li>
+             <li> <input type="text" class="name-email" name="email" placeholder="Email address" required></li>
```

Comment:
- [ ] Kindly ensure that all input fields have the correct `type` attribute. As a result, the email input's `type` equals `email` and not `text` :100: